corehq/apps/userreports/util.py

```diff
@@ -331,3 +332,9 @@ def wrap_report_config_by_type(config):
         }[config["doc_type"]].wrap(config)
     except KeyError:
         raise ReportConfigurationNotFoundError()
+
+
+def disallowed_ucr_expressions(domain_name):
```

Comment:
can this also be a `classmethod` for `AllowedUCRExpressionSettings`?

Reply:
Done https://github.com/dimagi/commcare-hq/pull/31161/commits/d35e6e0eca98b3d0c264e1a8e1ded3383e8f4177